src/backend.py

```diff
@@ -155,3 +156,11 @@ def handle_status_code(status_code):
     async def get_wow_character_achievements(self,  realm, character_name):
         url = f"https://{self._authentication_client.region}.api.blizzard.com/wow/character/{realm.lower()}/{character_name}?fields=achievements"
         return await self.do_request("GET", url)
+
+    async def get_ow_player_data(self):
+        url = 'https://owapi.io/profile/pc/' + self._authentication_client.region + '/' + self._authentication_client.user_details['battletag'].replace('#', '-')
+        response = await self.do_request('GET', url, None, False)
+        payload = json.loads(response.text)
+        if 'message' in payload: # user not found... unfortunately no 404 status code is returned :/
+            return None
```

Comment:
https://ow-api.com/v1/stats/pc/eu/ssaitho%232703/complete

gives me

```json
{
    "error": "Player not found"
}
```
So probably this needs to be changed to 
```python
if 'error' in playload:
    return None
```

> user not found... unfortunately no 404 status code is returned :/

We can still raise UnknownBackendResponse or something like that in this method if you feel its more appropriate, but IMO None is OK.

Reply:
The new URL now would be https://owapi.io/profile/pc/us/ssaitho-2703 which gives
`{"message":"Error: Profile not found"}`